src/components/Pages.tsx

```diff
@@ -0,0 +1,111 @@
+import "./Pages.scss";
+import Bacground1440 from "./icons/backgroundPages1440.svg";
+import Bacground360 from "./icons/backgroundPages360.svg";
+import Bacground744 from "./icons/backgroundPages744.svg";
+import Close from "./icons/close.svg";
+import { FC } from "react";
+import {Link} from "react-router-dom";
+import Logo1440 from "./icons/logo1440.svg";
+import Logo360 from "./icons/logo360.svg";
+import Logo744 from "./icons/logo744.svg";
```

Comment:
please, try to use background-property and try to move that .svg files to the scss

(maybe it's impossible, but at least - try)

Reply:
but it's bad practice to include images in css